Python/Product/PythonTools/PythonTools/Options/IPythonOptions.cs

```diff
@@ -55,6 +55,11 @@ public interface IPythonOptions {
             get;
             set;
         }
+
+        bool UseLegacyDebugger {
+            get;
+            set;
+        }
```

Comment:
This is a COM interface, so we need to add a new interface to expose the new option.